include/range/v3/detail/config.hpp

```diff
@@ -389,6 +389,14 @@ namespace ranges
 #define RANGES_RE_ENABLE_WARNINGS
 #endif
 
+
+#if __has_attribute(no_sanitize)
+#define RANGES_INTENDED_UNSIGNED_OVERFLOW \
```

Comment:
The pedant in you is wrong: unsigned types in C++ are closed under all operations; there are no operations that exceed the maximum representable value. Again, this is fundamentally what modulo behavior means. We are tagging the functions that actually want modulo behavior for unsigned integers, and leaving untagged those functions that wrongly use unsigned types to mean nonnegative.